packages/assertions/src/subtle-crypto.ts

```diff
@@ -21,11 +21,11 @@ async function isEd25519CurveSupported(subtle: SubtleCrypto): Promise<boolean> {
         cachedEd25519Decision = new Promise(resolve => {
             subtle
                 .generateKey('Ed25519', /* extractable */ false, ['sign', 'verify'])
-                .catch(() => {
-                    resolve((cachedEd25519Decision = false));
-                })
                 .then(() => {
                     resolve((cachedEd25519Decision = true));
+                })
```

Comment:
This was an actual bug that can cause unresolved rejections that blow up an app.